reporter/src/main/kotlin/reporters/AsciiDocsAttributionDocumentReporter.kt

```diff
@@ -32,7 +32,7 @@ import org.ossreviewtoolkit.reporter.utils.FreemarkerTemplateGenerator
 class AsciiDocsAttributionDocumentReporter : Reporter {
     companion object {
         private const val ASCIIDOCS_FILE_PREFIX = "Asciidocs_"
-        private const val ASCIIDOCS_DEFAULT_TEMPLATE = "asciidoc_test"
+        private const val ASCIIDOCS_DEFAULT_TEMPLATE = "asciidoc"
```

Comment:
This commit should be squashed into the previous commit, as the previous commit would otherwise not be functional, as "asciidoc_test" does not exist.